packages/synapse-react-client/src/components/GenericCard/Linkify.tsx

```diff
@@ -18,7 +18,15 @@ const synapseIdRule: LinkifyRule = {
 }
 
 const httpRule: LinkifyRule = {
```

Comment:
Fixes a linkify bug.  You can see for any secondary label value that was an URL, it autolinks but the previous separator resulted in an additional "https" matched token.